### PR TITLE
Introducing Putu Piring / Kueh Tutu

### PR DESCRIPTION
**Pull request by JuanitoFatas**
I was watching [Street Food](https://www.netflix.com/title/80244996), and this dish was introduced:

> It's got so much history. And when you mention it to anyone in Singapore, everyone knows about it, and then..."Oh, yeah. I used to eat it as a kid."

Then I checked this guide but not listed here. So here I am adding it haha. 

Anyone got a picture please put up ❤️

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@jhecking Thank you for fixed the typo 😄

**Comment by cheeaun** [comment]
Honestly spekaing, I've never tried Putu Piring before.

**Comment by JuanitoFatas** [comment]
> Honestly speaking, I've never tried Putu Piring before.

😱 Same here.

Wondering what do Singaporean here think about Putu Piring? 😅

**Comment by hanmd82** [comment]
I have never heard of 'Putu Piring' before - I know it by another name: 'kueh tutu'. Its pretty awesome and definitely a childhood favorite :) 👍

File: README.md [hunk 1]
```diff
@@ -330,6 +330,10 @@ if you're in Chinatown.
 
 ![Wanton Min](/images/wanton_min.jpg)
 
+### Putu Piring
```
Comment (hanmd82) [inline README.md]: suggest to make this `Putu Piring / Kueh Tutu`

**Pull request merged**